tools/ci_build/github/azure-pipelines/nuget/templates/test_win.yml

```diff
@@ -62,11 +62,6 @@ jobs:
     parameters:
       packageFolder: '$(Build.BinariesDirectory)\nuget-artifact'
 
-  - script: |
```

Comment:
should we keep this? Looks like it includes some more test models

Reply:
Let me double-check again. I remember w/wo this folder, the test code iterated on the same files.